core/vm/logger.go

```diff
@@ -124,6 +128,7 @@ func NewStructLogger(cfg *LogConfig) *StructLogger {
 	return logger
 }
 
+// CaptureStart logs the start of a contract
```

Comment:
// CaptureStart implements the Tracer interface to initialize the tracing operation.